service-loadbalancer/template.cfg

```diff
@@ -144,10 +144,11 @@ backend {{$svc.Name}}
     {{end}}
 {{end}}
 {{if and $svc.SessionAffinity $svc.CookieStickySession}}
+    # NOTE: Need to insert here too!
     # insert a cookie with name SERVERID to stick a client with a backend server
     # http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#4.2-cookie
     cookie SERVERID insert indirect nocache
-    {{range $j, $ep := $svc.Ep}}server {{$ep}} {{$ep}} cookie s{{$j}} check port {{$svc.BackendPort}} inter 5
+    {{range $j, $ep := $ip_set}}server {{$ep}} {{$ep}} cookie s{{$j}} check port {{$svc.BackendPort}} inter 5 agent-check agent-port 4001
```

Comment:
`agent-check` and `agent-port` need to be set via flags at the top of `service_loadbalancer.go`. Might need to be set on other entries if this is to be done correctly beyond this hack.